audio/src/libvorbis_decoder.rs

```diff
@@ -77,3 +72,9 @@ impl error::Error for VorbisError {
         error::Error::source(&self.0)
     }
 }
+
+impl From<vorbis::VorbisError> for AudioError {
```

Comment:
Here I'm struggling ... I thought it would be define in libvorbis because it does that for AudioDecoder. And I can't find a proper option if I define it lib.rs (help!)

Reply:
Unless I'm missing something, it should be possible to define

```
#[cfg(feature = "with-tremor")]
extern crate librespot_tremor as vorbis;
#[cfg(not(feature = "with-tremor"))]
extern crate vorbis;
```
at the top of `lib.rs`, and then further down by `AudioError`, convert from the `VorbisError` using a `From` implementation:

```
impl From<vorbis::VorbisError> for AudioError {
    fn from(err: vorbis::VorbisError) -> AudioError {
        AudioError::VorbisError(VorbisError::from(err))
    }
}
```
And then remove the existing implementations from the `libvorbis`/`lewton` decoders